test/integration/028_cli_vars/test_cli_vars.py

```diff
@@ -25,7 +25,9 @@ def test__cli_vars_longform(self):
             }
         }
         self.run_dbt(["run", "--vars", yaml.dump(cli_vars)])
-        self.run_dbt(["test"])
+
+        # TODO : This change in sensible, but why did it not fail before??
```

Comment:
did you figure it out? you should probably figure it out and remove this comment, or just remove it hah

Reply:
yeah -- this worked incorrectly before. The test should have failed, but the vars were stored in a global somewhere so it _looked_ like they were supplied again. I couldn't quite figure out why that happened, but can dig some more if you feel strongly about it.

It's only an issue since we're invoking dbt as a lib from the integration tests